src/commonMain/kotlin/exchange.dydx.abacus/calculator/AdjustIsolatedMarginInputCalculator.kt

```diff
@@ -0,0 +1,136 @@
+package exchange.dydx.abacus.calculator
+
+import exchange.dydx.abacus.output.input.IsolatedMarginAdjustmentType
```

Comment:
can use static imports for `Add` and `Remove` specifically to make the code easier to read below.
```
import exchange.dydx.abacus.output.input.IsolatedMarginAdjustmentType.Add
import exchange.dydx.abacus.output.input.IsolatedMarginAdjustmentType.Remove
```

Reply:
qq: wdym by easier to read? The import would be easier to read?

Reply:
Wow that's pretty cool! I don't have a preference on either though. With the static import I think we would lose some readability unless you double check the enum, and what happens if there are multiple enums with the same `.name`

Reply:
yeah up to you, i don't feel strongly. 

re multiple enums: you have to explicitly say what type you're casting the value to, so that ends up not being an issue. ex:
```
val type = "Add"
when(IsolatedMarginAdjustmentType.valueOf(type)) {
  Add -> ...
  Remove -> ...
}
```